htdocs/admin/ticket.php

```diff
@@ -219,7 +228,7 @@
 
 $dirmodels = array_merge(array('/'), (array) $conf->modules_parts['models']);
 
-$form = new Form($db);
+$form = new FormCategory($db);
```

Comment:
$form var is commonly used for   $form = new Form($db);
So can you rename the var into   $formcategory  ?